README.md

```diff
@@ -110,25 +110,26 @@ Checkout our [Errors documentation][errors] to learn more.
-| `client_secret` | String |**Required** Application clientSecret obtained from [Smartcar Developer Portal](https://dashboard.smartcar.com). |
-| `redirect_uri`  | String |**Required** RedirectURI set in [application settings](https://dashboard.smartcar.com/apps). Given URL must match URL in application settings. |
-| `scope`         | String[] |**Optional** List of permissions your application requires. This will default to requiring all scopes. The valid permission names are found in the [API Reference](https://smartcar.com/docs/api#get-all-vehicles). |
-| `test_mode`   | Boolean |**Optional** Launch the Smartcar Connect in test mode. |
+| Parameter       | Type | Required | Description   |
+|:--------------- |:---|:---|:------------- |
+| `client_id`     | String |**Required** | Application clientId obtained from [Smartcar Developer Portal](https://dashboard.smartcar.com). |
+| `client_secret` | String |**Required** | Application clientSecret obtained from [Smartcar Developer Portal](https://dashboard.smartcar.com). |
+| `redirect_uri`  | String |**Required** | RedirectURI set in [application settings](https://dashboard.smartcar.com/apps). Given URL must match URL in application settings. |
+| `scope`         | String[] |**Optional** | List of permissions your application requires. This will default to requiring all scopes. The valid permission names are found in the [API Reference](https://smartcar.com/docs/api#get-all-vehicles). |
+| `test_mode`   | Boolean |**Optional** | Launch the Smartcar Connect in test mode. |
 
 ### `get_auth_url(self, force=False, state=None, vehicle_info=None)`
 
 Generate the Connect URL
 
 #### Arguments
-| Parameter       | Type | Description   |
-|:--------------- |:---|:------------- |
-| `force`   | Boolean |**Optional** Setting `forcePrompt` to `true` will show the permissions approval screen on every authentication attempt, even if the user has previously consented to the exact scope of permissions. |
-| `state`         | String |**Optional** OAuth state parameter passed to the redirectUri. This parameter may be used for identifying the user who initiated the request. |
-| `vehicle_info['make']`  | String |**Optional** Including the dict `vehicle_info` with a `make` property allows users to bypass the car brand selection screen. For a complete list of supported makes, please see our [API Reference](https://smartcar.com/docs/api#authorization) documentation. Makes are case-insensitive. |
-
+| Parameter       | Type | Required | Description   |
+|:--------------- |:---|:---|:------------- |
+| `force`   | Boolean |**Optional** | Setting `forcePrompt` to `true` will show the permissions approval screen on every authentication attempt, even if the user has previously consented to the exact scope of permissions. |
+| `state`         | String |**Optional** | OAuth state parameter passed to the redirectUri. This parameter may be used for identifying the user who initiated the request. |
```

Comment:
OAuth state parameter passed to the **redirect URI**. This parameter may be used **to identify** the user who **has** initiated the request.